packages/@react-spectrum/s2/style/style-macro.ts

```diff
@@ -11,8 +11,13 @@
  */
 
 import type {Condition, CSSProperties, CSSValue, CustomValue, Property, PropertyValueDefinition, PropertyValueMap, RenderProps, ShorthandProperty, StyleFunction, StyleValue, Theme, ThemeProperties, Value} from './types';
+import fs from 'fs';
 import * as propertyInfo from './properties.json';
 
+// Postfix all class names with version for now.
+const json = JSON.parse(fs.readFileSync(__dirname + '/../package.json', 'utf8'));
+const POSTFIX = json.version.replace(/[0.]/g, '');
```

Comment:
just realized that this might break in nightly versions...

Reply:
oh good point, thats annoying. Can we hash the nightly version to a shortish string? Alternatively, maybe we could start appending a incrementing version number to the nightly version name and use that as part of the postfix (n1, n2, etc)?

Reply:
I think for nightly it's ok if it's a bit longer. I just made it include the hash in that case.